### ISSUE TITLE
Editorial question

### ISSUE THREAD

**Issue by xml-project** [xproc/3.0-specification#810]
Where do we say what we mean by "XSLT match pattern"? Did not find it. I think we should say we mean "XSLT 3.0" match pattern, not 2.0 match pattern.

**Comment by gimsieke** [comment]
There’s this list item before http://spec.xproc.org/master/head/xproc/#common-errors, introduced in https://github.com/xproc/3.0-specification/pull/424/files that fixes #424:
> …and the further requirement to be a conformant Expression per [XPath 3.1] or selection pattern per [XSLT 3.0]

The XSLT 3.0 spec seems to use “selection pattern” and “match pattern” interchangeably, without explicitly saying so. 
I suggest that we say at one point that we use both terms interchangeably, for example by adding:

> …or selection pattern per [XSLT 3.0]. As in the XSLT 3.0 specification, the colloquial term “match pattern” is used interchangeably with “selection pattern” in this specification.

While researching this, I stumbled over [this](http://spec.xproc.org/master/head/xproc/#psvi-support.10.4.1.1): 
> If an implementation supports XPath 2.0 or later, the data model constructed with which to evaluate XPath expressions and match patterns should take advantage of as much PSVI information as possible.

Since XPath 3.1 is mandatory now, we should probably rephrase this to: “The data model constructed with which to evaluate XPath expressions and match patterns should take advantage of as much PSVI information as possible.”

**Comment by gimsieke** [comment]
And we should probably tag “selection pattern” and “match pattern” as `glossterm`s. But then we’d also need a corresponding `termdef` I guess.
